demo-simple/src/main/java/com/novoda/downloadmanager/demo/CustomNotificationConfigFactory.java

```diff
@@ -0,0 +1,50 @@
+package com.novoda.downloadmanager.demo;
+
+import android.app.Notification;
+import android.content.Context;
+import android.support.v4.app.NotificationCompat;
+import android.support.v4.app.NotificationManagerCompat;
+
+import com.novoda.downloadmanager.DownloadBatchStatus;
+import com.novoda.downloadmanager.DownloadBatchTitle;
+import com.novoda.downloadmanager.NotificationConfig;
+import com.novoda.downloadmanager.NotificationCustomiser;
+import com.novoda.notils.logger.simple.Log;
+
+public final class CustomNotificationConfigFactory {
```

Comment:
This class is unused, it replaces the functionality from the now-deleted `CustomNotificationCreator`.